executor/executor_test.go

```diff
@@ -29,6 +25,9 @@ import (
 	"github.com/pingcap/tidb/util/testkit"
 	"github.com/pingcap/tidb/util/testleak"
 	"github.com/pingcap/tidb/util/types"
+	"strings"
+	"testing"
+	"time"
```

Comment:
We put go standard library packages above other packages, with an empty line to separate them.